### DISCUSSION TITLE
ubanble to install properly, i did every thing but its showing in analytics, i have no idea whats wrong any help

### DISCUSSION THREAD

**Discussion by bikashamit** [invertase/react-native-firebase#4715] [Q&A]
ubanble to install properly, i did every thing but its showing in analytics, i have no idea whats wrong any help.

**Comment by mikehardy** [comment]
Is this a report of an actual problem?

Imagine I was your co-worker and I came to your desk and said: "ubanble to install properly, i did every thing but its showing in analytics, i have no idea whats wrong any help."

...how exactly would you help?

I can say if you want to see a full install demonstration you can check every command in this script, and run it, and you will see how to install. https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh

**Reply by bikashamit** [reply]
its problem, i tried like 20 times but every time i didn't get any error. but after installation it should have shown in analytics but it does not shows up there.

**Reply by mikehardy** [reply]
"it" is a problem, what is "it" exactly?
"I didn't get any error" -> did you expect an error? Exactly what error were you expecting?
"after installation" -> installation of what? Perhaps you could show a reproducible demo or run my demo script?
"should have shown in analytics" -> what should have shown in analytics?

I'm sorry, this is so vague I cannot help. I will not respond to this further. If you have a problem please log a new issue following the template and posting an App.js we can drop into the demo script above that reproduces the problem.

Please note analytics is one of our most used modules. We know for certain it works. So pay very very close attention to the install instructions and re-test everything from the assumption the module works, but your project is incorrect - we won't be able to help with project-specific items unless you can show the module itself has a problem
